src/app/app.routes.ts

```diff
@@ -4,28 +4,33 @@ import { LoginComponent } from './pages/login';
 import { AddCourseComponent } from './pages/courses/addCourse';
 import { NoContentComponent } from './no-content';
 import { DataResolver } from './app.resolver';
+import { PageNotFoundComponent } from './pages/pageNotFound';
 
 export const ROUTES: Routes = [
   {
-    path: '',
+    path: 'courses',
```

Comment:
tutaj klika scieżek ma ten sam początek `'courses'`, także mógłbyś to przerobić coś w stylu:
```
  {path: '', redirectTo: '/courses', pathMatch: 'full' },
	{path: 'login', component: LoginComponent},
	{
		path: 'courses',
		component:[SomeBaseEmptyComponent],
		children: [
			{
				path: '',
				component: HomeComponent
			},
			{
				path: 'new',
				component: AddCourseComponent
			},
			{
				path: ':id',
				component: AddCourseComponent,
				data: {
          type: 'edit'
        }
			}
		]
	},
```